spec/spec_helper.rb

```diff
@@ -18,7 +19,7 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 
-
+Coveralls.wear!('rails')
```

Comment:
Do we know the performance implications of including this line in our test suite?